emukit/examples/vanilla_bayesian_quadrature_with_rbf/vanilla_bq_loop_with_rbf.py

```diff
@@ -4,44 +4,50 @@
 
 import numpy as np
 import GPy
-from typing import List
+from typing import List, Tuple, Union
 
 from emukit.quadrature.methods.vanilla_bq import VanillaBayesianQuadrature
 from emukit.quadrature.loop import VanillaBayesianQuadratureLoop
-from emukit.model_wrappers.gpy_quadrature_wrappers import QuadratureRBF, RBFGPy, BaseGaussianProcessGPy
+from emukit.quadrature.kernels.integration_measures import IntegrationMeasure
+from emukit.model_wrappers.gpy_quadrature_wrappers import convert_gpy_model_to_emukit_model
 
 
-def create_vanilla_bq_loop_with_rbf_kernel(X: np.ndarray, Y: np.ndarray, integral_bounds: List,
-                                           rbf_lengthscale: float=1.0, rbf_variance: float=1.0) -> \
-        VanillaBayesianQuadratureLoop:
+def create_vanilla_bq_loop_with_rbf_kernel(X: np.ndarray, Y: np.ndarray,
+                                           integral_bounds: Union[None, List[Tuple[float, float]]],
+                                           measure: Union[None, IntegrationMeasure],
+                                           rbf_lengthscale: float=1.0, rbf_variance: float=1.0) \
+        -> VanillaBayesianQuadratureLoop:
     """
 
     :param X: initial training point locations, shape (n_points, input_dim)
     :param Y:  initial training point function values, shape (n_points, 1)
     :param integral_bounds: List of input_dim tuples, where input_dim is the dimensionality of the integral
     and the tuples contain the lower and upper bounds of the integral i.e.,
-    [(lb_1, ub_1), (lb_2, ub_2), ..., (lb_D, ub_D)].
+    [(lb_1, ub_1), (lb_2, ub_2), ..., (lb_D, ub_D)]. None means infinite bounds.
+    :param measure: the integration measure. None means no measure.
     :param rbf_lengthscale: the lengthscale of the rbf kernel, defaults to 1.
     :param rbf_variance: the variance of the rbf kernel, defaults to 1.
     :return: The vanilla BQ loop
     """
 
-    if not len(integral_bounds) == X.shape[1]:
-        D_bounds = len(integral_bounds)
-        input_dim = X.shape[1]
-        raise ValueError("number of integral bounds " + str(D_bounds) + " provided does not match the input dimension "
-                                                                        + str(input_dim) + ".")
+    if integral_bounds is not None:
+        if not len(integral_bounds) == X.shape[1]:
+            D_bounds = len(integral_bounds)
+            input_dim = X.shape[1]
+            raise ValueError("number of integral bounds ", D_bounds, " provided does not match the input "
+                             "dimension ", input_dim, ".")
```

Comment:
I am entirely sure what way of constructing the error message does. Is the assumption here that all values will be concatenated? 

Even if it is, a clearer way would be to use `"".format` or string interpolation

There are many places in this CR where the comment applies

Reply:
done